docs/usage/cli.rst

```diff
@@ -121,15 +121,15 @@ entries should point to a :class:`click.Command` or :class:`click.Group`:
         .. code-block:: toml
             :caption: Using `poetry <https://python-poetry.org/>`_
 
-            [tool.poetry.plugins."litestar.commands"]
+            [project.entry-points."litestar.commands"]
             my_command = "my_litestar_plugin.cli:main"
 
     .. tab-item:: uv
 
         .. code-block:: toml
             :caption: Using `uv <https://docs.astral.sh/uv/>`_
 
-            [project.scripts]
+            [project.entry-points."litestar.commands"]
```

Comment:
Since this has been standardised across tools, these individual tool-specific sections are unnecessary and should just be removed entirely